guides/release-process.md

```diff
@@ -67,7 +67,13 @@ of Cypress. You can see the progress of the test projects by opening the status
 
 ![Screenshot of status checks](https://i.imgur.com/AsQwzgO.png)
 
-Once the `develop` branch for all test projects are reliably passing with the new changes, publishing can proceed.
+#### :bangbang: Important :bangbang:
+
+The `linux x64`, `win32 x64`, and `darwin x64` artifacts produced by CI are all placed in the same directory on the CDN. The version that was built last will overwrite the other versions in the directory. Until work is done to complete [#19771](https://github.com/cypress-io/cypress/issues/19771), you must ensure that the `linux` workflow publishes its artifacts **after** the `windows`/`mac` workflows. To guarantee this, you can re-run the `linux` workflow within CircleCI by selecting the 'Rerun workflow from start' option for the `linux` workflow after the initial builds have completed.
```

Comment:
Technically you wouldn't have to re-run the entire workflow to get around this. You can restart the [`create-build-artifacts` job](https://app.circleci.com/pipelines/github/cypress-io/cypress/30138/workflows/24ea861b-c405-49a9-ba11-f042d1a3d4d6/jobs/1167411) by selecting re-run job with SSH to re-upload to the CDN. This would be a lot faster.

<img width="417" alt="Screen Shot 2022-01-21 at 3 45 59 PM" src="https://user-images.githubusercontent.com/14099737/150604353-6ec2dd50-93be-4860-9899-f23c20ae7f12.png">